tests/test_images.py

```diff
@@ -0,0 +1,61 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+from geograypher.utils.image import perspective_from_equirectangular
+
+
+@pytest.mark.parametrize(
+    "expected,yaw_deg,pitch_deg",
+    (
+        [[(45, 270), (90, 315), (135, 270), (90, 225)], 90, 0],
+        [[(45, 0), (90, 45), (135, 0), (90, 315)], 180, 0],
```

Comment:
Probably a good idea to test some pitch and roll cases as well just for completeness.